src/common/snippets/include/snippets/op/perf_count.hpp

```diff
@@ -11,6 +11,69 @@
 
 namespace ov {
 namespace snippets {
+
+namespace op {
+class PerfCountEnd;
+} // namespace op
+
+namespace utils {
+
+/**
+ * @interface Dumper
+ * @brief Dumper for node debug properties
+ * @ingroup snippets
+ */
+class Dumper {
+public:
+    Dumper() = default;
+    virtual ~Dumper() = default;
+
+    void init(const std::string &params);
+    virtual void update(const op::PerfCountEnd* node) = 0;
+protected:
+    std::map<std::string, std::string> m_debug_params_map;
+    std::string m_params;
+};
+
+/**
+ * @interface ConsoleDumper
+ * @brief Dumper for node debug properties (output: stdout)
+ * @ingroup snippets
+ */
+class ConsoleDumper : public Dumper {
+public:
+    ConsoleDumper() = default;
+    ConsoleDumper(const ConsoleDumper&) = delete;
```

Comment:
Here and in CSVDumper: why do you [keep deleting](https://github.com/openvinotoolkit/openvino/pull/28378#discussion_r1935284289) copy constructors? 🙃 
There must be a reason for it, could you share?